l10n_br_base/models/res_partner.py

```diff
@@ -35,6 +33,25 @@ class Partner(models.Model):
 
     union_entity_code = fields.Char(string="Union Entity code")
 
+    cnpj_cpf = fields.Char(
```

Comment:
@JorgeJuarezCasai 

I tested this PR changes on runbot, in res.partner it's fine, but if you change the vat in a res.company, it is not updated in company's related partner.

To solve this you can move this changes to https://github.com/OCA/l10n-brazil/blob/14.0/l10n_br_base/models/party_mixin.py#L17 because party.mixin is injected in res.partner and res.company